Userland/Libraries/LibGfx/ImageFormats/PortableFormatWriter.cpp

```diff
@@ -37,12 +37,13 @@ ErrorOr<void> PortableFormatWriter::add_pixels(Stream& output, Options const& op
         auto row = TRY(FixedArray<u8>::create(bitmap.width() * 3ul));
 
         for (int i = 0; i < bitmap.height(); ++i) {
-            for (int j = 0; j < bitmap.width(); ++j) {
-                auto color = bitmap.get_pixel(j, i);
+            auto const width = bitmap.width();
```

Comment:
This could probably be moved outside entirely if we are pedantic. Same with a variable for `height()`, assuming that this makes a noticeable performance difference in the first place.

Reply:
Let's not bother with that, I deleted it!